app.rb

```diff
@@ -71,4 +73,69 @@ def list_all_labels
       end
     end
   end
+
+  def add_music_album
+    print 'Is the Music Album on Spotify? [Y/N]: '
+    is_spotify = gets.chomp.downcase
+    on_spotify = true if on_spotify?(is_spotify)
+    on_spotify = false unless on_spotify?(is_spotify)
+    puts on_spotify
+    print 'What\'s the publishing date? [year/month/day] (e.g 1937/11/12): '
+    published_date = gets.chomp
+    print 'What\'s the label title of the music? '
+    label_title = gets.chomp
+    print 'What\'s the label color / studio of the music? '
+    color = gets.chomp
+    print 'What\'s the genre of the music? '
+    genre_name = gets.chomp
+    puts "\n \n Music Album created successfully \n \n"
+    save_music_album(on_spotify, published_date, label_title, color, genre_name)
+  end
+
+  def on_spotify?(is_spotify)
+    case is_spotify
+    when 'y'
+      true
+    when 'n'
+      false
+    else
+      puts 'Invalid Selection. Please try again!'
+      exit
```

Comment:
Great work making a helper function, will the `exit` here quit from the app, kindly check that and confirm

You can do a recursion like this
```
def on_spotify?
    print 'Is the Music Album on Spotify? [Y/N]:'
    is_spotify = gets.chomp.downcase

    case is_spotify
    when 'y'
      true
    when 'n'
      false
    else
      puts 'Invalid selection. kindly enter \'y\', \'Y\' or \'n\', \'N\' 😀'
      on_spotify?
    end
  end
```